ui/src/App.js

```diff
@@ -1,21 +1,46 @@
 import React, { Component } from 'react';
+import axios from 'axios';
 import logo from './logo.svg';
 import './App.css';
 
 class App extends Component {
-  render() {
-    return (
-      <div className="App">
-        <div className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h2>Welcome to React</h2>
-        </div>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
-      </div>
-    );
-  }
+	state = {
+		info: null,
+		error: null
+	}
+
+	componentDidMount() {
+		axios.get('/info')
+			.then((response) => {
+				this.setState({
+					info: response,
+					error: null,
+				});
+			})
+			.catch((error) => {
+				this.setState({
+					error: error.message
+				});
+			});
+	}
+
+	render() {
+		const { info, error } = this.state;
+		return (
+			<div className="App">
```

Comment:
Just out of curiosity, is it "normal" to capitalize class names?  Maybe it's just me, but I always lowercased.